src/main/java/com/ollymonger/dynmap/portals/DynmapPortalsPlugin.java

```diff
@@ -126,20 +127,34 @@ public void onPortalCreate(PortalCreateEvent event) throws IOException {
     }
 
     private void writePortals() {
-        try{
+        try {
             Path path = Paths.get(getDataFolder().getAbsolutePath(), "portals.json");
             File file = new File(path.toString());
             file.getParentFile().mkdirs();
             OutputStreamWriter os = new OutputStreamWriter(new FileOutputStream(file), "UTF-8");
 
             os.write(gson.toJson(this.registeredPortals, registeredPortalListType));
+            getLogger().info("portal done");
 
             os.close();
         } catch (IOException e) {
             e.printStackTrace();
         }
     }
 
+    private void loadPortals() {
+        try {
+            Path path = Paths.get(getDataFolder().getAbsolutePath(), "portals.json");
+            File file = new File(path.toString());
+            JsonReader reader = new JsonReader(new FileReader(file));
+            List<RegisteredPortal> data = gson.fromJson(reader, registeredPortalListType);
+
+            getLogger().info(String.valueOf(data.size()));
```

Comment:
Remove this log or make it more full (e.g. "Loaded 5 portals from portals.json")